### DISCUSSION TITLE
CD Ripping EXTREMELY Slow

### DISCUSSION THREAD

**Discussion by slydog75** [automatic-ripping-machine/automatic-ripping-machine#1213] [General]
I installed ARM a few days ago and ripped one CD in relatively normal time (maybe 10 minutes). however, today I go to rip another one, and it says it's going to take nearly 4 days.  I'm running ARM in an unRAID docker.  When I look at the CPU usage, it's randomly pegging one single core or thread at a time.  

Any ideas what's going on? What can I supply to help?

**Comment by JollyRgrs** [comment]
I've found that abcde has had some issues reading certain discs. I believe due to whatever "copy protection" and/or enhanced CD features they have. Also had one with issues due to a "negative space" track (rewind from the start of track 1 and you can have negative time).

I am using iSCSI to connect my USB BD-ROM drive to ARM. I used the same iscsi target to connect it to my Windows PC and the same disc ripped fine with EAC (though there was usually a 1st or last track that was not coming through as a music track).

Can you pull the logs from that disc? I assume it will have a bunch of lines about illegal something or another. The logs and also if you can provide any details on the disc (if it is an enhanced CD, etc.). I'm curious about that. 

Also... this being said, this isn't something much that could be controlled by ARM (I'm not any dev at all, so please don't take my word as a final thing). I'd say this, from ARM's perspective, is an ABCDE issue. And inside of the ABCDE world, this is the process by cdparanoia... so in the end, that's really where the issue might lie. But again, the log will help us confirm that this is an issue on that end and not something else.
